test/dlc_tests/container_tests/bin/transformerengine/testPTTransformerEngine

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+git clone --branch release_v0.12 https://github.com/NVIDIA/TransformerEngine.git
+cd TransformerEngine/tests/pytorch
+
+pip install pytest==6.2.5 onnxruntime==1.13.1 onnx
+pytest -v -s test_sanity.py
+PYTORCH_JIT=0 NVTE_ALLOW_NONDETERMINISTIC_ALGO=0 pytest -v -s test_numerics.py
+NVTE_TORCH_COMPILE=0 pytest -v -s test_onnx_export.py
+pytest -v -s test_jit.py
```

Comment:
good to add `set -ex` https://github.com/aws/deep-learning-containers/blob/master/test/dlc_tests/container_tests/bin/efa/testEFA#L4